src/getShopifyVariables.ts

```diff
@@ -5,6 +5,7 @@ const getShopifyVariables = (
 ): Required<SanityQueryClientOptions['shopifyVariables']> => {
   return {
     // Defaults taken from https://shopify.dev/beta/hydrogen/getting-started#step-4-make-graphql-changes
+    country: 'US',
```

Comment:
@benjaminsehl I lack the context on Shopify's API, but couldn't defaulting to US be risky to international users with a single non-US country? Perhaps we could adapt `useSanityShopifyProducts`' query to only include `@inContext` if country is defined?

Reply:
Yeah that's a great idea, I'm on board with that. I'll have to look into how I go about that — feel free to make the edit if you want.

Reply:
I definitely agree with you. 
In your suggestions below, where does the `country` variable come from?